src/UnityExtension/Assets/Editor/GitHub.Unity/UI/PublishView.cs

```diff
@@ -35,7 +39,7 @@ class PublishView : Subview
         [NonSerialized] private IApiClient client;
```

Comment:
Not directly related to this line, but some lines above the `IList<Organization>` field needs fixing. It isn't serializable by Unity, it needs to be an instance of a thing. Also, `Organization` is probably not very serializable either, given what it contains, so we probably need a type of our own for this and serialize it as `List<Thing>`